collect_app/build.gradle

```diff
@@ -216,10 +216,6 @@ android {
         htmlReport true
         lintConfig file("$rootDir/config/lint.xml")
         xmlReport true
-
-        if (!project.hasProperty("lintStrings")) {
```

Comment:
This workflow makes it much easier to just add strings quickly in a PR (as copy becomes less of a concern) so I don't think we need this.